### DISCUSSION TITLE
npx prisma studio not working

### DISCUSSION THREAD

**Discussion by Slimshadynick** [prisma/prisma#7715] [Q&A]
Error: listen EAFNOSUPPORT: address family not supported :::5555. I am getting this error when running above command.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
